Trellix/trellix-epo-on-prem/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "value": "Threat detected on {host.ip}",
+    "conditions": [
+
+    ]
+  }
+]
```

Comment:
Can you add conditions to this smart-description? I think we can have one smart-description for EPOEvents and another one for UpdateEvents.